articles/communication-services/how-tos/chat-sdk/data-loss-prevention.md

```diff
@@ -40,8 +40,7 @@ chatClient.on("chatMessageEdited", (e) => {
 ```javascript
 const messages = chatThreadClient.listMessages();
 for await (const message of messages) {
-    if (message.content?.message == "" &&
-        message.sender?.kind == "microsoftTeamsUser") {
+    if (message.policyViolation.result == "contentBlocked") {
```

Comment:
I think this needs a null check, policy violation is probably not always set:

message.policyViolation?.result

Reply:
Good point, updated